static/open-specs/chat.yaml

```diff
@@ -108,22 +108,26 @@ components:
         reactions:
           type: object
           description: Represents a summary of all reactions on a message
-          properties:
-            unique:
-              type: object
-              description: Represents a summary of all unique reactions on a message
-              additionalProperties:
-                $ref: '#/components/schemas/SummaryClientIdList'
-            distinct:
-              type: object
-              description: Represents a summary of all distinct reactions on a message
-              additionalProperties:
-                $ref: '#/components/schemas/SummaryClientIdList'
-            multiple:
-              type: object
-              description: Represents a summary of all multiple reactions on a message
-              additionalProperties:
-                $ref: '#/components/schemas/SummaryClientIdCounts'
+          $ref: '#/components/schemas/MessageReactions'
+    MessageReactions:
+      type: object
+      description: Represents a summary of all reactions on a message
+      properties:
+        unique:
+          type: object
+          description: Represents a summary of all unique reactions on a message
+          additionalProperties:
+            $ref: '#/components/schemas/SummaryClientIdList'
+        distinct:
+          type: object
+          description: Represents a summary of all distinct reactions on a message
```

Comment:
This is perhaps a misleading comment and muddies the waters between this and unique (distinct reactions, unique reactions - sounds like the same thing).

Can we relate it more directly to the summary schemes / behaviour?

Reply:
I've rephrased. I can change them to more closely match the descriptions in [this table](https://ably.com/docs/chat/rooms/message-reactions#types-of-reactions) but this is a reference so not sure how long to make it?